README.md

```diff
@@ -31,7 +48,9 @@ None
   roles:
     - ansible-role-vim
   vars:
-    vim_additional_packages: []
+    # XXX FreeBSD does not have _small_, and vim-related packages. to be tested
```

Comment:
it has `sysutils/vimpager`, or it is not necessarily a vim-related one for testing.